tests/index.test.ts

```diff
@@ -1,13 +1,8 @@
-import { exec } from 'child_process';
-import { existsSync } from 'fs';
-import { resolve } from 'path';
-import util from 'util';
+import { existsSync } from 'node:fs';
+import { resolve } from 'node:path';
```

Comment:
This is only available as of Node v18

```suggestion
import { existsSync } from 'fs';
import { resolve } from 'path';
```

Reply:
I think it's available since v16, but still out of the range we support 🤦 . I'm reverting it!